### PR TITLE
Fix/mailchimp sync

### PR DESCRIPTION
**Pull request by luizalbsilva**
Sincronização na primeira utilização

### CODE REVIEW

File: app/resque_jobs/mailchimp_sync.rb [hunk 1]
```diff
@@ -19,32 +19,32 @@ def self.create_segment_if_necessary(widget)
...
-		activistMatch = ActivistMatch.find(activist_pressure_id)
-		widget = activistMatch.widget 
-		if ( widget )
+	def self.perform_with_activist_match(activist_match_id)
+		activist_match = ActivistMatch.find(activist_match_id)
+		widget = activist_match.widget 
+		if ( widget ) and ( widget.id ) and ( not activist_match.synchronized )
 			self.create_segment_if_necessary(widget)
-			activistMatch.update_mailchimp
+			activist_match.update_mailchimp
```
Comment (lpirola) [inline app/resque_jobs/mailchimp_sync.rb]: não existe esse tipo de checagem para doacão?

File: app/models/widget.rb [hunk 2]
```diff
@@ -3,29 +3,34 @@ class Widget < ActiveRecord::Base
...
 
-  after_create :async_create_mailchimp_segment
   delegate :user, to: :mobilization
 
   def as_json(*)
     WidgetSerializer.new(self, {root: false})
   end
 
   def segment_name
+    kinds_correlation = {'pressure' => 'P', 'form' => 'F', 'match' => 'M'}
```
Comment (lpirola) [inline app/models/widget.rb]: maybe is missing donation kind?

Comment (luizalbsilva) [inline app/models/widget.rb]: Donation is not a Mailchimpable -- it's not synchronized wih mailchimp. Will we turn it a mailchimp synchronizable entity?

**Comment by lpirola** [comment]
@luizalbsilva we must continue to follow the above rules:

* Do not allow empty segments;
* Do not register segments from widget with type different from actions defined;
* Create segment when new subscriber are added;
* Create group in mailchimp after create community;
* set group from mailchimp in subscriber to allow more than one organization by user;

**Comment by luizalbsilva** [comment]
I'm working the creation of new groups on another branch, you may merge this one if you like.

**Pull request merged**